contentcuration/contentcuration/views/files.py

```diff
@@ -236,6 +241,72 @@ def exercise_image_upload(request):
     }))
 
 
+@authentication_classes((TokenAuthentication, SessionAuthentication))
+@permission_classes((IsAuthenticated,))
+def subtitle_upload(request):
```

Comment:
Nice! We've been talking about automatic file conversion on upload for a while now, so it's great to see it here. A couple thoughts here: 
- It would be great if this endpoint could be used more broadly (see comment above). I was thinking you could use the extension to determine special cases or something
- This conversion code seems like it could be useful for the content pipeline as well. Do you mind moving this to a `converter.py` file under the [pressurecooker](https://github.com/learningequality/pressurecooker/tree/master/pressurecooker)?

Reply:
I created the `multilanguage_file_upload` endpoint. Also, I have already integrated the conversion handling into the pipeline.

Reply:
Awesome, thank you!